rakelib/task_helpers.rb

```diff
@@ -10,7 +10,7 @@ def examples_dir
       File.join(gem_root, "gen-examples", "examples")
     end
 
-    def dummy_app_dirs
+    def dummy_app_dir
```

Comment:
this was technically dead code so it was okay for me to change it here and not break anything